german.feskov/task-2-1/cmd/service/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/6ermvH/german.feskov/task-2-1/internal/temperature"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	bufChanSize = 1
+)
+
+func main() {
+	var countDep int
+
+	if _, err := fmt.Scan(&countDep); err != nil {
+		fmt.Println(err)
+
+		return
+	}
+
+	for range countDep {
+		var countWorker int
+		if _, err := fmt.Scan(&countWorker); err != nil {
+			fmt.Println(err)
```

Comment:
написать бы тут конечно детали сообщений, например:

```suggestion
			fmt.Printf("failed to read count of workers: %v\n", err)
```